### PR TITLE
NEW COURSES ARE ADDED.

### PR DESCRIPTION
**Pull request by Ashu104**
## What does this PR do? 
Add resource(s) 
### Description
new free courses are added 
### Why is this valuable (or not)?
complete courses are available for free of cost.
### How do we know it's really free?
we can visit them on study for zero cost.
### For book lists, is it a book? For course lists, is it a course? etc.
added both a book and some courses
## Checklist:
- [x] Read our [contributing **guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).**
- [ ] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -359,6 +359,7 @@
 ### Cloud Computing
 
 * [Cloud Computing and Development](https://uaceit.com/courses/cloud-computing-and-development) - Ashwin Kumar Ramaswamy (UAceIt) (email address *required*)
+* [Cloud Computing course](https://www.youtube.com/playlist?list=PLShJJCRzJWxhz7SfG4hpaBD5bKOloWx9J) - nptel course
```
Comment (eshellman) [inline courses/free-courses-en.md:362]: Don't invent titles! Use the title given by the resource.
```suggestion
* [NPTEL-CloudComputing](https://www.youtube.com/playlist?list=PLShJJCRzJWxhz7SfG4hpaBD5bKOloWx9J) - Manish Narula
```

File: courses/free-courses-en.md [hunk 2]
```diff
@@ -421,9 +422,12 @@
 
 ### Cryptography
 
+* [Cryptography & Network Security](https://www.youtube.com/playlist?list=PLBlnK6fEyqRgJU3EsOYDTW7m6SUmW6kII) - Neso Academy
+* [Foundations of Cryptography](https://www.youtube.com/playlist?list=PLgMDNELGJ1CbdGLyn7OrVAP-IKg-0q2U2) - nptel course
```
Comment (eshellman) [inline courses/free-courses-en.md:426]:
```suggestion
* [Foundations of Cryptography](https://www.youtube.com/playlist?list=PLgMDNELGJ1CbdGLyn7OrVAP-IKg-0q2U2) - NPTEL  Indian Institute of Science, Bengaluru
```

File: courses/free-courses-en.md [hunk 3]
```diff
@@ -421,9 +422,12 @@
...
 * [Introduction to Cryptography](https://open.ruhr-uni-bochum.de/en/lernangebot/introduction-cryptography) (Christof Paar)
 * [Stanford Cryptography I](https://www.coursera.org/course/crypto) - Dan Boneh
 * [Stanford Cryptography II](https://www.coursera.org/course/crypto2) - Dan Boneh
+* [Undertanding Cryptography](https://www.youtube.com/playlist?list=PL2jrku-ebl3H50FiEPr4erSJiJHURM9BX) - Dr Christof Paar
```
Comment (eshellman) [inline courses/free-courses-en.md:430]: Don't invent titles! Use the title given by the resource.
```suggestion
* [Cryptography Lecture Series](https://www.youtube.com/playlist?list=PL2jrku-ebl3H50FiEPr4erSJiJHURM9BX) - Christof Paar
```

File: courses/free-courses-en.md [hunk 4]
```diff
@@ -508,6 +512,7 @@
 * [Learn SQL](https://www.sololearn.com/learning/1060) - *registration required*
 * [Learn SQL: SQL Tutorial for Beginners](https://www.programiz.com/sql) - Programiz
 * [MySQL Tutorial for Beginners [Full Course]](https://www.youtube.com/watch?v=7S_tz1z_5bA) - Moshfegh Hamedani (Programming with Mosh)
+* [SQL complete course](https://www.youtube.com/playlist?list=PLLQPIumE5cE-gzU5hChH1V3H93x4UOlHR) - nptel course
```
Comment (eshellman) [inline courses/free-courses-en.md:515]:
```suggestion
* [SQL NPTEL](https://www.youtube.com/playlist?list=PLLQPIumE5cE-gzU5hChH1V3H93x4UOlHR)
```
Don't invent titles! Use the title given by the resource. Even if it sucks.

**Review by eshellman — CHANGES_REQUESTED** [review]
some of these aren't very good. Please exercise some curatorial discretion.

**Comment by Ashu104** [comment]
hi @eshellman. 
glad you checkout my pr and pointing out the mistakes and issues.  I appreciate your feedback and will carefully consider your suggestions in order to enhancing my skills to contribute better in this project.

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**